internal/connector/test/integration/features/connector-agent-api.feature

```diff
@@ -1114,6 +1114,33 @@ Feature: connector agent API
     }
     """
 
+    #-----------------------------------------------------------------------------------------------------------------
```

Comment:
Would be nice to add steps to test patching with a json body containing different fields that are not patchable and expect and error.

Reply:
@dhirajsb explicitly asked for the tests to be mixed in between your scenario. You guys make up your mind and let me know.

Reply:
> Would be nice to add steps to test patching with a json body containing different fields that are not patchable and expect and error.

+1